sdk/spring/spring-cloud-azure-service/src/main/java/com/azure/spring/service/eventhubs/properties/EventBatchProperties.java

```diff
@@ -3,6 +3,8 @@
 
 package com.azure.spring.service.eventhubs.properties;
 
+import com.azure.spring.service.implementation.eventhubs.properties.EventProcessorClientProperties;
+
```

Comment:
empty line

Reply:
There supposed to be empty line between the com.xxx and java.xxx